kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -77,7 +78,8 @@ class ExecuteStatement(
         ms.incCount(STATEMENT_OPEN)
         ms.incCount(STATEMENT_TOTAL)
       }
-      _remoteOpHandle = client.executeStatement(statement, shouldRunAsync, queryTimeout)
+      // always run in async mode on remote engine
```

Comment:
The code here already tells `always run in async mode on remote engine`. So let's comment not only just WHAT but also WHY

Reply:
Updated